syntax = "proto3";

message PSPullVec
{
    repeated uint64 keys = 1;
    int32 timestamp = 2;
}